src/tokenizer.rs

```diff
@@ -1086,8 +1079,8 @@ impl<'a> Tokenizer<'a> {
     }
 
     /// Tokenize an identifier or keyword, after the first char is already consumed.
-    fn tokenize_word(&self, first_chars: String, chars: &mut State) -> String {
-        let mut s = first_chars;
+    fn tokenize_word(&self, first_chars: impl Into<String>, chars: &mut State) -> String {
```

Comment:
Uses some generics to avoid having to manually construct `String`s in several places.